src/Kdyby/ElasticSearch/Diagnostics/Panel.php

```diff
@@ -104,6 +104,11 @@ public function getPanel()
 				return array();
 			}
 
+			if ($object instanceof Elastica\Request) {
+				$j = new Elastica\JSONFormat('  ', "\n"); // indent and linebreak characters
```

Comment:
Where is this class from? I can't find it https://github.com/ruflin/Elastica/search?l=php&q=Format&utf8=%E2%9C%93